ext/standard/password.c

```diff
@@ -202,7 +202,7 @@ static zend_string* php_password_bcrypt_hash(const zend_string *password, zend_a
 	ZSTR_VAL(salt)[ZSTR_LEN(salt)] = 0;
 
 	hash = zend_string_alloc(ZSTR_LEN(salt) + hash_format_len, 0);
-	sprintf(ZSTR_VAL(hash), "%s%s", hash_format, ZSTR_VAL(salt));
+	snprintf(ZSTR_VAL(hash), ZSTR_LEN(hash) + 1, "%s%s", hash_format, ZSTR_VAL(salt));
 	ZSTR_VAL(hash)[hash_format_len + ZSTR_LEN(salt)] = 0;
```

Comment:
Can't this use `zend_strpprintf`?

Reply:
Yes, or even better, zend_string_concat2, I'll change it to the latter.